Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt

```diff
@@ -10,8 +10,8 @@
     	"type": "typo3-cms-extension",
     	"description": "Example site package from the site package tutorial",
     	"require": {
-    		"typo3/cms-core": "^13.4.0|dev-main",
-    		"typo3/cms-fluid-styled-content": "^13.4.0|dev-main"
+    		"typo3/cms-core": "^13.3.0|dev-main",
```

Comment:
Why change to 13.3?

Reply:
This file is auto-generated, the problem lies in the site package extension